.github/workflows/fullsync-tests.yml

```diff
@@ -129,10 +129,17 @@ jobs:
       run: |
         chmod 777 defid
         chmod 777 defi-cli
-        chmod 777 sync_then_diff.sh
+        chmod 777 sync.sh
 
-    - name: Sync and Diff
-      run: ./sync_then_diff.sh
+    - name: Sync
+      run: ./sync.sh
+
+    - name: Diff log
+      run: diff debug-${{matrix.blocks.stop}}.log debug-tmp-${{matrix.blocks.stop}}.log
+
+    - name: Diff rollback log
+      run: diff debug-pre-rollback.log debug-post-rollback.log
+      if: ${{ failure() ||  success() }}
```

Comment:
Need this?

Reply:
Not strictly needed but let's it run even if we have a failure on previous diff log.
It let's us have both diff results but will still mark the sync as failed from previous diff failure (whereas `continue-on-error` would mark it as passed).